protos/camera_server/camera_server.proto

```diff
@@ -234,6 +234,28 @@ enum CameraFeedback {
     CAMERA_FEEDBACK_FAILED = 3; // Failed
 }
 
+/*
+ * Camera capability flags as defined in MavLink: 
+ * https://mavlink.io/en/messages/common.html#CAMERA_CAP_FLAGS
+ *
+ * Multiple status fields can be set simultaneously. Mavlink does
+ * not specify which states are mutually exclusive.
+ */
+message CameraCapFlags {
```

Comment:
CameraCapFlags is not independent message, so use enum type is more clear.

Reply:
Tanks for the comment. However, this piece of code s already removed in the newest code version